02_configure_host.sh

```diff
@@ -156,7 +156,7 @@ ansible-playbook \
 # NOTE(elfosardo): /usr/share/OVMF/OVMF_CODE.fd does not exist in the ovmf
 # package anymore, so we need to create a link to that until metal3-dev-env
 # fixes that, probably when switching to UEFI by default
-if ! [[ -f /usr/share/OVMF/OVMF_CODE.fd ]]; then
+if ! [[ -e /usr/share/OVMF/OVMF_CODE.fd ]]; then
```

Comment:
this works, maybe also -L or a combination of the two?

Reply:
Done.